ministryofjustice/correspondence_tool_staff
# CT-2675 Added ico-appeal sections to cabinet rep

## Description

this PR is to add extra sections for ICO-appeal to meet the needs of annual-ico-appeal report

## Self-review checklist


### Screenshots

After
<img width="500" alt="Screenshot 2020-09-28 at 13 21 34" src="https://user-images.githubusercontent.com/26111671/94437058-28ba8780-0195-11eb-8258-755191a5bd8c.png">

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=CT&modal=detail&selectedIssue=CT-2675
### Deployment


### Manual testing instructions

1- Login as London user
2- Click "Reports" from to menu
3- Click Cabinet office report
4- Check the figure from the report
